src/Fhir.Anonymizer.Stu3.Core.UnitTests/ConfigurationVersionTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Fhir.Anonymizer.Core.AnonymizerConfigurations;
+using Fhir.Anonymizer.Core.Extensions;
+using Hl7.FhirPath;
+using Xunit;
+
+namespace Fhir.Anonymizer.Core.UnitTests.AnonymizerConfigurations
+{
+    public class ConfigurationVersionTests
+    {
+        public ConfigurationVersionTests()
+        {
+            FhirPathCompiler.DefaultSymbolTable.AddExtensionSymbols();
+        }
+
+        public static IEnumerable<object[]> GetInvalidVersionOfConfigs()
+        {
+        
+            yield return new object[] { "./TestConfigurationsVersion/configuration-R4-version.json" };
+            yield return new object[] { "./TestConfigurationsVersion/configuration-invalid-version.json" };
+
+        }
+
+        public static IEnumerable<object[]> GetValidVersionOfConfigs()
+        {
+            yield return new object[] { "./TestConfigurationsVersion/configuration-Stu3-version.json" };
+            yield return new object[] { "./TestConfigurationsVersion/configuration-empty-version.json" };
+            yield return new object[] { "./TestConfigurationsVersion/configuration-null-version.json" };
+        }
+
+        [Theory]
+        [MemberData(nameof(GetInvalidVersionOfConfigs))]
+        public void GivenAnInvalidVersion_WhenCreateAnonymizerConfigurationManager_ExceptionShouldBeThrown(string configFilePath)
+        {
+            Assert.Throws<AnonymizerConfigurationErrorsException>(() => AnonymizerConfigurationManager.CreateFromConfigurationFile(configFilePath));
```

Comment:
We can use validator.validate() in unit tests directly, as the code change is in validator.